app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_save :update_post_like_counter
+
+  private
+
+  def update_post_like_counter
+    post.update_like_counter
```

Comment:
- [ ] I could not see any method in post model named `update_like_counter`, so it looks like you are calling a method that does not exist,  kindly add it and its necessary logics